draft-ietf-tls-tls13.md

```diff
@@ -3429,6 +3431,35 @@ flight, the server MUST be prepared to receive an arbitrary number
 of such messages before receiving the Authentication messages.
 
 
+#### Key Update 
+
+struct {} KeyUpdate;
+
+The KeyUpdate handshake message is used to indicate that the sender is
+updating its sending cryptographic keys. This message can be sent by
+the server after sending its first flight and the client after sending
+its second flight. Implementations prior to this time MUST generate a
```

Comment:
"Endpoints that receive a KeyUpdate message prior to receiving a Finished message MUST generate a fatal ..."